appveyor.yml

```diff
@@ -0,0 +1,29 @@
+
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - ARCH:                x64
+      D_COMPILER:          dmd
+      D_VERSION:           2.077.1
```

Comment:
We shouldn't forget to change this to 2.078.0 once released and remove the hard-coded dmd-nightly.